src/shared/components/chevron-button/ChevronButton.scss

```diff
@@ -0,0 +1,6 @@
+@import 'src/styles/common';
+
+.chevronDisabled {
+  --chevron-fill: $light-grey;
+  cursor: default;
```

Comment:
I thought it will be the default behaviour of a disabled button anyway; but it seems, a button doesn't even make the cursor change into a pointer on hover when it is not disabled. Should cursor: pointed be added as a default style?

Reply:
I've added ` cursor: pointer` to Chevron's wrapper class.